cmd/operator-sdk/bundle/cmd.go

```diff
@@ -50,16 +50,16 @@ https://sdk.operatorframework.io/docs/olm-integration/
 func NewCmdLegacy() *cobra.Command {
 	cmd := newCmd()
 	cmd.AddCommand(
-		newCreateCmd(),
-		newValidateCmd(),
+		NewCreateCmd(),
+		NewValidateCmd(),
```

Comment:
Why we need to export the NewCreateCmd and NewValidateCmd?

Reply:
So i can call them from the tests.